src/scenes/MembershipMintPage/CustomizedGeneralMembershipMintPage.tsx

```diff
@@ -0,0 +1,363 @@
+import { Web3Provider } from '@ethersproject/providers';
```

Comment:
this is essentially a copied version of the more generic [MembershipMintPage](https://github.com/gallery-so/gallery/blob/main/src/scenes/MembershipMintPage/MembershipMintPage.tsx)

while i was initially working in that file, it became a nightmare to wrangle all kinds of states, since the flow / edge cases for Early Access was slightly different than silver/gold. as a result i just made a new file.

the downside is this makes it less DRY; the upside is we'll rarely use these components after minting is complete